dist/env-webext.js

```diff
@@ -0,0 +1,9 @@
+const auth0Env = {
```

Comment:
I think this file should probably not be committed, it should be in protected config and then be copied in by the npm task

Reply:
ah right, I will fix. thanks!